### PR TITLE
[ipa-4-9] ipatests: Update PR-CI definitions for ipa-4-9

### PR DESCRIPTION
**Pull request by netoarmando**
Adding PR-CI definitions for gating and "latest" nightly runs.

Not including a definition for previous release, this must be done when updating from 32 to 33.

Signed-off-by: Armando Neto <abiagion@redhat.com>

### CODE REVIEW

**Comment by flo-renaud** [comment]
- `test_cert` and `test_forced_client_enrolment` failed in unintall with DBus error, known failure https://pagure.io/freeipa/issue/8506
- `test_acme` is a known failure https://pagure.io/freeipa/issue/8581

**Comment by netoarmando** [comment]
@flo-renaud @rcritten Could the failure in `test_acme.TestACMEwithExternalCA::test_mod_md` be a regression related to https://pagure.io/freeipa/issue/8581?

**Comment by flo-renaud** [comment]
@netoarmando it looks like a different problem. On the master branch, even with the recent fixes I saw another issue, #[8598](https://pagure.io/freeipa/issue/8598). The test is not stable yet...

**Review by flo-renaud — COMMENTED** [review]
Hi @netoarmando 
Thanks for the PR. Please find my comments:
- The file ipatests/prci_definitions/nightly_ipa-4-9_latest.yaml is missing the test test_installation_client.py that was recently added with commit https://github.com/freeipa/freeipa/commit/82e45d1f1234c472f235d9ca8922387fff3bb2e0
- Same comment for ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml
Otherwise LGTM.

**Comment by flo-renaud** [comment]
An additional comment: since on ipa-4-8 branch there is a PR updating from f32 to f33, you could also do the same in this PR.

**Comment by netoarmando** [comment]
@flo-renaud thanks for catching the missing test. 

> An additional comment: since on ipa-4-8 branch there is a PR updating from f32 to f33, you could also do the same in this PR.

Done. I also added a temp commit to test gating tests with Fedora 33.

Once this is merged I'll enable the automation to the all runs.

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 1]
```diff
@@ -41,1759 +41,1639 @@ topologies:
...
+        template: *ci-ipa-4-9-previous
         timeout: 7200
         topology: *master_1repl
 
-  testing-fedora/test_membermanager:
-    requires: [testing-fedora/build]
-    priority: 50
+  fedora-previous-ipa-4-9/test_membermanager:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 100
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1539]: usually tests are run with priority: 50, was it a mistake?

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 2]
```diff
@@ -41,1759 +41,1639 @@ topologies:
...
+        template: *ci-ipa-4-9-previous
         timeout: 7200
         topology: *master_1repl
 
-  testing-fedora/test_adtrust_install:
-    requires: [testing-fedora/build]
-    priority: 50
+  fedora-previous-ipa-4-9/test_adtrust_install:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 100
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1599]: usually tests are run with priority: 50, was it a mistake?

**Review by flo-renaud — COMMENTED** [review]
Hi,
I spotted 2 differences but they can be fixed later.

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 2 above)
Comment (netoarmando) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1599]: I've copied the content from master branch's `nightly_latest.yaml`, no idea why there were set with `100` there. 
PR-CI will execute them before the rest of the tests, that would be the only "side effect".

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 1 above)
Comment (netoarmando) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1539]: Same as https://github.com/freeipa/freeipa/pull/5280#discussion_r559686065: 
> I've copied the content from master branch's nightly_latest.yaml, no idea why there were set with 100 there.
PR-CI will execute them before the rest of the tests, that would be the only "side effect".

**Comment by flo-renaud** [comment]
LGTM, the priorities can be changed in a different PR.

**Comment by flo-renaud** [comment]
ipa-4-9:

* ccdecaa984ef6ebcc63d754e896b2229bcba3b88 ipatests: Update PR-CI definitions for ipa-4-9

**Pull request closed without merging**